admin/scripts/modules/infopult.xtpl

```diff
@@ -159,10 +159,12 @@
         <!-- end:covidSekce -->
 
         <tr>
-          <td colspan="100%" style="clearfix">
-            <input style="float: left;" type="checkbox" class="" name="kontrolaOsobnichUdaju" {kontrolaOsobnichUdajuAttr}>
-            <div style="float: left;">{kontrolaOsobnichUdajuText}</div>
-            <input style="float: right;" type="submit" value="uložit" class="submit-btn">
+          <td colspan="100%">
```

Comment:
`style="clearfix"` byl nějaký copy paste z reactu? Firefox to nepobral. A fungovalo to i bez toho, tak jsem to smazal.